contracts/upgradeable_contracts/erc20_to_erc20/ForeignBridgeErcToErc.sol

```diff
@@ -22,6 +19,29 @@ contract ForeignBridgeErcToErc is BasicBridge, BasicForeignBridge {
         uint256 _homeMaxPerTx,
         address _owner
     ) public returns(bool) {
+        _initialize(
+            _validatorContract,
+            _erc20token,
+            _requiredBlockConfirmations,
+            _gasPrice,
+            _maxPerTx,
+            _homeDailyLimit,
+            _homeMaxPerTx,
+            _owner
+        );
+        return isInitialized();
+    }
+
+    function _initialize(
+        address _validatorContract,
+        address _erc20token,
+        uint256 _requiredBlockConfirmations,
+        uint256 _gasPrice,
+        uint256 _maxPerTx,
```

Comment:
Do you have any idea why we left `_maxPerTx` here? If we look at `validations` in `loadEnv.js`, there is no check for this parameter in case of `erc20-to-erc20` bridge that's why I think it was left here by mistake.

Reply:
I think I found the reason. It was introduced as part of this issue for `erc-to-native` and `erc-to-erc` modes https://github.com/poanetwork/poa-bridge-contracts/issues/110. So if it is correct that `_maxPerTx` should be there, then we need to add the validation on `loadEnv.js`

Reply:
Oh! Thanks for finding this

Reply:
Added validation for `maxPerTx` for all bridge modes https://github.com/poanetwork/poa-bridge-contracts/pull/177/commits/17b907a821c4a4f6f3f9589597f1b5c581f577d9